docs/src/main/tut/optiont.md

```diff
@@ -51,6 +51,33 @@ val noWelcome: OptionT[Future, String] = customGreetingT.filterNot(_.contains("w
 val withFallback: Future[String] = customGreetingT.getOrElse("hello, there!")
 ```
 
+## From `Option[A]` and/or `F[A]` to `OptionT[F, A]`
+
+Sometimes you may have an `Option[A]` and/or `F[A]` and want to *lift* them into an `OptionT[F, A]`. For this purpose `OptionT` exposes two useful methods, namely `fromOption` and `liftF`, respectively. E.g.:
+
+```tut:silent
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import cats.data.OptionT
+import cats.std.future._
```

Comment:
I can see the benefit in self-contained examples. My only concern is that currently with tut I don't think we have a way to clear previous imports in the same file. So we run the risk of duplicating imports without actually guaranteeing that the example _is_ self-contained. I don't have any really strong feelings on this though, so whatever you think is best.

Regarding the misleading missing implicit value message:
This came up in the gitter channel [yesterday](https://gitter.im/non/cats?at=564a3a658b242470793e3053). I don't particularly like any of the solutions that we have in mind, but if there's something we can do to help, then we should. If you want to file an issue for that, that would be great!